lib/winapi/xapi.c

```diff
@@ -5,9 +5,44 @@
  */
 
 #include <xapi.h>
+#include <assert.h>
 #include <stdio.h>
+#include <string.h>
 #include <xboxkrnl/xboxkrnl.h>
 
+__attribute__((constructor)) static VOID setup_d_drive (VOID)
+{
+    NTSTATUS status;
+    HANDLE handle;
+    ANSI_STRING driveString = RTL_CONSTANT_STRING("\\??\\D:");
+    OBJECT_ATTRIBUTES objattr;
+
+    InitializeObjectAttributes(&objattr, &driveString, OBJ_CASE_INSENSITIVE, NULL, NULL);
+    status = NtOpenSymbolicLinkObject(&handle, &objattr);
+    if (NT_SUCCESS(status)) {
+        NtClose(handle);
+        return;
+    }
```

Comment:
This could be its own function to test if a drive is mounted?

Reply:
Done.